### DISCUSSION TITLE
I don't know why my account was suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#28845] [Discussions]
I don't know why my account was suspended. Can anyone please help me? thanks
I wonder if it was because I sent out my wechat yesterday.I know this time, please give me a chance! I will be very grateful! I am a newbie! I didn't read the rules carefully, it was my fault! I'm very sorry! Rest assured, I won't be breaking the rules in the future! Because I think my Github is important to me!
By the way, can I write my own email address here? so staff can contact me

**Comment by ghost** [comment]
This is my new account, I want to get back my old account

**Comment by melvincwng** [comment]
@ChetSerenade09 , Hi there, for account-related issues, my suggestion is to raise a support ticket to [GitHub Support](https://support.github.com/contact/account) over here and let the support engineers investigate the issue.

It may take a few working days (up to a few weeks) depending on their investigation.

Hope this clarifies.

**Reply by ghost** [reply]
Thanks!

**Reply by melvincwng** [reply]
@ChetSerenade09, no problemo, glad to have helped!

If this reply has been helpful, please kindly 'Mark as answer' so the rest are aware this issue has been addressed.

Thanks, and have a good day ahead 😄
